extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -357,7 +357,12 @@ export abstract class AzureAuth implements vscode.Disposable {
 		const tenantUri = url.resolve(this.metadata.settings.armResource.endpoint, 'tenants?api-version=2019-11-01');
 		try {
 			Logger.verbose('Fetching tenants', tenantUri);
-			const tenantResponse = await this.makeGetRequest(tenantUri, token.token);
+			Logger.verbose('Tenant request token: ', token.token, []);
```

Comment:
The token is very sensitive information and should not be something we log lightly - this should definitely be a PII logging statement at least. And then I would further go and only log a part of the token (just to verify it's there and looks valid). Or do you think you need the full token for something?

Reply:
Yes I need the full token for this customer